.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+$finder = (new PhpCsFixer\Finder())
+    ->in([
+        __DIR__ . '/src',
+        __DIR__ . '/inc',
+    ])
```

Comment:
When everyone will be OK with the config, this inclusion list will have to be reviewed to include ajax, front, tests, etc... files.

Reply:
:+1:

Reply:
I've added other directories, and run fix on it, to see the full changelog.